### DISCUSSION TITLE
Tuya

### DISCUSSION THREAD

**Discussion by moamen121** [milo526/homebridge-tuya-web#320] [Q&A] [answered]
Help my friend 
https://user-images.githubusercontent.com/75143496/130917487-b3ddb65e-84ca-47ce-be81-6d5896821598.MOV

**Comment by milo526** [comment, marked as answer]
Works as intended, this is a limitation by Tuya that I cannot bypass.
In other words, I cannot fix this problem.

**Reply by moamen121** [reply]
So what I do has no solution for me🤦‍♂️😟

**Reply by milo526** [reply]
Correct

**Reply by moamen121** [reply]
disappointed😕 ok thanks 🙏

**Reply by milo526** [reply]
Just to get back to this, the slightly longer answer is that Tuya does not give a signal to my plugin indicating that the switch has changed. The plugin needs to ask Tuya, "Has anything changed?"
It can only do so once every 5 minutes.
You'll see that the state is correctly reflected after just over 5 minutes (at the latest).
